packages/mattermost-redux/src/actions/channel_categories.ts

```diff
@@ -291,13 +290,6 @@ export function moveChannelToCategory(categoryId: string, channelId: string, new
             return {error};
         }
 
-        // Update the favorite preferences locally on the client in case we have any logic relying on that
```

Comment:
The web app no longer uses prefs to determine if a channel is favourited, so this all could be removed.